kodi_addon_checker/check_dependencies.py

```diff
@@ -2,26 +2,31 @@
 from .report import Report
 from .record import PROBLEM, Record, WARNING
 
+common_ignore_deps = ['xbmc.metadata.scraper.albums', 'xbmc.metadata.scraper.albums', 'xbmc.metadata.scraper.movies',
+                      'xbmc.metadata.scraper.musicvideos', 'xbmc.metadata.scraper.tvshows',
+                      'xbmc.metadata.scraper.library', 'xbmc.ui.screensaver', 'xbmc.player.musicviz',
+                      'xbmc.python.pluginsource', 'xbmc.python.script', 'xbmc.python.weather', 'xbmc.python.lyrics',
+                      'xbmc.python.library', 'xbmc.python.module', 'xbmc.subtitle.module', 'kodi.context.item',
+                      'kodi.game.controller', 'xbmc.gui.skin', 'xbmc.webinterface', 'xbmc.addon.repository',
+                      'xbmc.pvrclient', 'kodi.gameclient', 'kodi.peripheral', 'xbmc.addon.video',
+                      'xbmc.addon.audio', 'xbmc.addon.image', 'xbmc.addon.executable', 'kodi.addon.game',
+                      'kodi.audioencoder', 'kodi.audiodecoder', 'xbmc.service', 'kodi.resource.images',
+                      'kodi.resource.language', 'kodi.resource.uisounds', 'kodi.resource.games',
+                      'kodi.resource.font', 'kodi.inputstream', 'kodi.vfs', 'kodi.imagedecoder', 'xbmc.json',
+                      'xbmc.gui', 'xbmc.json', 'xbmc.metadata', 'xbmc.python', 'script.module.pil',
+                      'inputstream.adaptive', 'script.module.pycryptodome']
```

Comment:
shouldn't be added here

Reply:
https://github.com/xbmc/addon-check/commit/862de90c1ba3b7c1afff310af92802979cb32bb0

Reply:
`script.module.pycryptodome` is also wrong here